app/scripts/proactive/templates/file-browser-template.html

```diff
@@ -1,4 +1,4 @@
-<div class="modal-dialog">
+<div class="modal-dialog" style="width:60vw">
```

Comment:
why not add this to the class `modal-style` in css if it applies to all modal-class divs ?

Reply:
This one is specific to the file browser modal now. (It requires a larger width as it has more info in each line)

Reply:
can use bootstrap class .modal-sm or .modal-lg ?

Reply:
https://getbootstrap.com/docs/4.2/components/modal/